content/2017.06.kinto-elasticsearch.rst

```diff
@@ -0,0 +1,249 @@
+Kinto and ElasticSearch
+#######################
+
+:lang: en
+:date: 2017-06-21
+:url: kinto-elasticsearch
+:summary: Introducing our ElasticSearch plugin for Kinto
+
+
+ElasticSearch integration
+=========================
+
+Kinto is a generic API on top of a storage backend.
+ElasticSearch is a generic API on top of an indexing backend. When combining both,
+your data is stored in a persistent database, and you can search, aggregate and
+filter it efficiently.
+
+This integration has been sitting around for a long time, since it was the matter of
+our `Kinto plugin tutorial <http://kinto.readthedocs.io/en/7.0.1/tutorials/write-plugin.html>`_! But recently, we had a concrete use-case at Mozilla
```

Comment:
I'm not sure an exclamation mark is necessary here :) Maybe move it to the next sentence.